src/components/ErrorPage/ErrorPage.js

```diff
@@ -62,14 +62,18 @@ const ErrorPage = ({
   errorMessage =
     !errorMessage && errorLabels ? errorLabels.ERRORMESSAGE : errorMessage;
 
+  let formattedBackgroundImage;
+
   if (!backgroundImage) {
-    backgroundImage = errorIllustrations[statusCode]
+    formattedBackgroundImage = errorIllustrations[statusCode]
       ? { ...errorIllustrations[statusCode] }
       : { ...errorIllustrations.default };
+  } else {
+    formattedBackgroundImage = { backgroundImage };
   }
 
   return (
-    <section className={classes} style={backgroundImage} {...other}>
+    <section className={classes} style={formattedBackgroundImage} {...other}>
```

Comment:
In the process of adding a test for `backgroundImage` prop, I realized that custom background images are not put in the correct `style` attribute format (i.e., inside an object).